net/tailscale/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+tailscale -version | grep "$2"
+tailscaled -version | grep "$2"
```

Comment:
I suggest testing if `tailscale` / `tailscaled` exists before running it, since the CI will install each ipk separately. (Can also check if at least one is installed, if you want to get fancy.)

Reply:
Although, since tailscale depends on tailscaled, when the tailscale ipk is installed by opkg, tailscaled (the buildbot-built package, not the ipk built during the CI run) will also be installed but it will not have the correct version information, so the test will still fail (for this PR, should pass in the future).

Reply:
The current tailscaled in the feed returned `1.24.2-dev` in the [test](https://github.com/openwrt/packages/runs/6881214373?check_suite_focus=true#step:14:41):

```
Testing package tailscale in version 1.24.2 from tailscale
Installing tailscale (1.24.2-2) to root...
2022/06/14 13:37:16 logtail started
2022/06/14 13:37:16 Program starting: v1.24.2-dev-t, Go 1.18.3: []string{"/usr/sbin/tailscaled", "--cleanup"}
2022/06/14 13:37:16 LogID: f17db0ab8cc78d82059348224cf738fedea0137f6e14d646bbc0e2418c509f20
2022/06/14 13:37:16 logpolicy: using UserCacheDir, "/root/.cache/Tailscale"
logpolicy.Read /root/.cache/Tailscale/tailscaled.log.conf: open /root/.cache/Tailscale/tailscaled.log.conf: no such file or directory
2022/06/14 13:37:16 dns: resolvedIsActuallyResolver error: resolv.conf doesn't point to systemd-resolved; points to [168.63.129.16]
2022/06/14 13:37:16 dns: [rc=resolved resolved=not-in-use ret=direct]
2022/06/14 13:37:16 dns: using *dns.directManager
2022/06/14 13:37:16 flushing log.
2022/06/14 13:37:16 logger closing down
Failed to connect to ubus
Installing kmod-tun (5.10.120-1) to root...
Downloading https://downloads.openwrt.org/snapshots/targets/x86/64/kmods/5.10.120-1-596cd7216a51a1d449b810129b5cce13/kmod-tun_5.10.120-1_x86_64.ipk
Installing tailscaled (1.24.2-1) to root...
Downloading https://downloads.openwrt.org/snapshots/packages/x86_64/packages/tailscaled_1.24.2-1_x86_64.ipk
Configuring kmod-tun.
Configuring tailscaled.
Configuring tailscale.
Use package specific test.sh
/usr/sbin/tailscale
1.24.2
/usr/sbin/tailscaled
1.24.2-dev
Test successful
Removing package tailscale from root...
```